.github/workflows/check_on_different_r_os.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - {os: macOS-latest,   r: 'release'}
           - {os: windows-latest, r: 'release'}
           - {os: ubuntu-20.04,   r: 'devel', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-18.04,   r: 'oldrel',  rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest"}
+          - {os: ubuntu-20.04,   r: 'oldrel',  rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
```

Comment:
So in fact you are doing twice the same check?
What I always wondered: why do you run exactly the same check on different versions of the same OS?  To truly check a package (and not just if its dependencies still run on the old version of the OS), I think it would be more useful for the 'oldrel' to check the package for older versions of the dependencies, for instance the versions that were new versions at the moment of the previous R update at INBO.

Reply:
Those checks are still different. `devel` refers to the daily build of R (will become `R 4.x+1`). Whereas `oldrel` refer to the previous stable minor version of R (`R 4.x-1.latest`). People no longer should be using Ubuntu 18.04. Hence there is no reason to test on 18.04. In the near future we will test `devel` on Ubuntu 22.04 and `oldrel` on Ubuntu 20.04.